### PR TITLE
Install jq in the Docker image, in the right layer

### PR DESCRIPTION
**Pull request by fiunchinho**
I thought I couldn't screw something so simple but I somehow managed to do it. So fixing it.

## Checklist

- [X] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,10 +13,10 @@ RUN apk add --no-cache ca-certificates curl jq \
 FROM quay.io/giantswarm/alpine:3.11-giantswarm as base
 
 USER root
-RUN apk add --no-cache git
+RUN apk add --no-cache git jq
 
 USER giantswarm
 COPY --from=binaries /binaries/* /usr/bin/
-COPY ./standup /usr/local/bin/standup
+COPY ./standup-linux /usr/local/bin/standup
```
Comment (fiunchinho) [inline Dockerfile:20]: it wouldn't work without this fix

Comment (stone-z) [reply Dockerfile:20]: Why did the binary name change?

Comment (stone-z) [reply Dockerfile:20]: This fails on CircleCI now
```
COPY failed: stat /var/lib/docker/tmp/docker-builder262920037/standup-linux: no such file or directory
```

Comment (fiunchinho) [reply Dockerfile:20]: I believe there is a miss-match between what the Makefile does, and what CircleCI does. `standup-linux` is the name of the binary generated by the Makefile. But it's not the one used by CircleCI. I'll revert the binary name change.

Comment (fiunchinho) [reply Dockerfile]: PTAL

**Review by stone-z — APPROVED** [review]

**Pull request merged**